tests/test_tv.py

```diff
@@ -19,31 +19,24 @@ def test_tv_works(prediction: torch.Tensor) -> None:
     measure = total_variation(prediction, size_average=True, reduction_type='l2')
     assert measure > 0
     measure = total_variation(prediction, size_average=True, reduction_type='l1')
+    assert measure > 0
     measure = total_variation(prediction, size_average=True, reduction_type='l2_squared')
+    assert measure > 0
 
 
-def test_tvloss_init(prediction: torch.Tensor, target: torch.Tensor) -> None:
+def test_tvloss_init() -> None:
     TVLoss()
 
 
-def test_tvloss(prediction: torch.Tensor, target: torch.Tensor) -> None:
+def test_tvloss(prediction: torch.Tensor) -> None:
     loss = TVLoss()
-    res = loss(prediction, target)
+    res = loss(prediction)
     assert res > 0
 
 
-def test_tv_zero_for_equal_tensors(prediction: torch.Tensor):
-    loss = TVLoss()
-    target = prediction.clone()
-    measure = loss(prediction, target)
-    assert measure <= 1e-6, f'TV for equal tensors must be 0, got {measure}'
-
-
 def test_tv_for_known_answer():
     # Tensor with `l1` TV = (10 - 1) * 2  * 2 = 36
     prediction = torch.eye(10).reshape((1, 1, 10, 10))
-    # Tensor with TV = 0
-    target = torch.zeros((1, 1, 10, 10))
     loss = TVLoss(reduction_type='l1')
-    measure = loss(prediction, target)
+    measure = loss(prediction)
     assert measure == 36., f'TV for this tensors must be 36., got {measure}'
```

Comment:
I think this test will break when changes are incorporated. Since the purpose of this test is to verify correctness of computations, please introduce another value that you are confident in or compare the result with some other trustworthy implementation.

Reply:
I checked the tests from `test_tv.py` on my machine and all of them were passed successfully

Reply:
@snk4tr btw, how to turn on the Actions pipeline?

Reply:
@smikhai1 first, check that Actions are not disabled in your fork of the repo. If they are, try to enable them. Note that testing and linting pipelines are triggered with new pushed commits 
<img width="730" alt="image" src="https://user-images.githubusercontent.com/24959314/82754561-3cfd0b00-9dd6-11ea-9773-e1e14abb416d.png">